feat(tab-nav-drawer, tabs): Disable animation. #7157 (#8255)
IgniteUI/igniteui-angular: projects/igniteui-angular/src/lib/navigation-drawer/README.md

| `pinThreshold` | number | Minimum device width required for automatic pin to be toggled. Default is 1024, can be set to a falsy value to disable this behavior. |
| `width` | string| Width of the drawer in its open state. Defaults to "280px".|
| `miniWidth` | string | Width of the drawer in its mini variant. Defaults to "60px". |

> | `disableAnimation` | boolean | Enables/disables the animation for the  opening and the closing of the drawer. Set to `false` by default.

Reviewer: ```suggestion
| `disableAnimation` | boolean | Enables/disables the animation, when toggling the drawer. Set to `false` by default.
```

Author: Changed as suggested.
